src/ReinforcementLearningBase/src/interface.jl

```diff
@@ -404,6 +404,15 @@ Make an independent copy of `env`,
 @api copy(env::AbstractEnv) = deepcopy(env)
 @api copyto!(dest::AbstractEnv, src::AbstractEnv)
 
+# checking the state of all players in env is enough?
+"""
+    Base.:(==)(env1::T, env2::T) where T<:AbstractEnv
+!!! warning: Only check the state of all players in the env.
```

Comment:
```suggestion
!!! warning
    Only check the state of all players in the env.
```

This is the correct format for adding `warning`. Please read https://docs.julialang.org/en/v1/stdlib/Markdown/#Admonitions for more details.